content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Change Notes
+description: Istio 1.8 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** DNS capture in istio-agent by default for VMs installed using `istioctl x workload entry configure`.
+
+- **Added** support for injecting `istio-cni` into `k8s.v1.cni.cncf.io/networks` annotation with preexisting value that uses JSON notation.
+  ([Issue #25744](https://github.com/istio/istio/issues/25744))
+
+- **Added** support for `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` insert operations for `HTTP_ROUTE` in `EnvoyFilter`  ([Issue #26692](https://github.com/istio/istio/issues/26692))
+
+- **Added** `REPLACE` operation for `EnvoyFilter`. `REPLACE` operation can replace the contents of a named filter with new contents. It is only valid for `HTTP_FILTER` and `NETWORK_FILTER`.
+  ([Issue #27425](https://github.com/istio/istio/issues/27425))
+
+- **Added** Istio resource status now includes observed generation
+  ([Issue #28003](https://github.com/istio/istio/issues/28003))
+
+- **Fixed** remove endpoints when the new labels in `WorkloadEntry` do not match the `workloadSelector` in `ServiceEntry`.
+  ([Issue #25678](https://github.com/istio/istio/issues/25678))
```

Comment:
This can use some re-wording. I don't quite understand this fix so if someone explains it to me I can help in the phrasing.